Tests/SwiftGenKitTests/ColorsTextFileTests.swift

```diff
@@ -6,47 +6,28 @@
 
 import XCTest
 import PathKit
-import SwiftGenKit
+@testable import SwiftGenKit
 
 class ColorsTextFileTests: XCTestCase {
-  func testEmpty() {
-    let parser = ColorsTextFileParser()
-
-    let result = parser.stencilContext()
-    XCTDiffContexts(result, expected: "empty.plist", sub: .colors)
-  }
-
-  func testListWithDefaults() throws {
-    let parser = ColorsTextFileParser()
-
-    try parser.addColor(named: "Text&Body Color", value: "0x999999")
-    try parser.addColor(named: "ArticleTitle", value: "#996600")
-    try parser.addColor(named: "ArticleBackground", value: "#ffcc0099")
-
-    let result = parser.stencilContext()
-    XCTDiffContexts(result, expected: "entries.plist", sub: .colors)
-  }
-
   func testFileWithDefaults() throws {
-    let parser = ColorsTextFileParser()
-    try parser.parseFile(at: Fixtures.path(for: "colors.txt", sub: .colors))
+    let parser = ColorsFileParser()
+    parser.colors = try ColorsTextFileParser().parseFile(at: Fixtures.path(for: "colors.txt", sub: .colors))
 
     let result = parser.stencilContext()
```

Comment:
Yeah might not be worth the hassle of maintaining all of that just for making tests more unitary. Maybe we can re discuss that in a separate PR.